### ISSUE TITLE
GitHub Sponsors

### ISSUE THREAD

**Issue by mrexodia** [frang75/nappgui_src#6]
Could you set up GitHub Sponsors, this project is really cool and I would like to support it.

**Comment by frang75** [comment]
Hi @mrexodia!

Thank you very much for your offer to be a sponsor. I was not aware of the ability to create sponsors on GitHub for the project. NAppGUI has been OpenSource for a very short time. 

I'll keep it in mind.

**Comment by mrexodia** [comment]
Yeah you can create a FUNDING.yml with various options: https://github.com/build-cpp/cmkr/blob/main/.github/FUNDING.yml

Just beware that it might take some time to get approved for GitHub sponsors so you might want to sign up sooner rather than later :)

**Comment by Hirrolot** [comment]
Damn nice project. Please, consider adding some options for sponsoring, not necessarily GitHub Sponsors. Maybe Patreon or just a PayPal account link would suffice.

Keep doing great work, @frang75! In the near future, I would like to sponsor this project too.

**Comment by frang75** [comment]
Hi @Hirrolot!
Thank you very much for your trust in the NAppGUI project. I am evaluating sponsorship alternatives for the near future. For now, any kind of testing and feedback would be helpful. 

We are in contact.
